rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy

```diff
@@ -3062,6 +3066,7 @@ If false, do not import the project configuration (default).''', schema = @Schem
 If false, do not import the ACL Policies (default).''', schema = @Schema(implementation = Boolean.class)),
                     @Parameter(name = 'importScm ', required = false, in = ParameterIn.QUERY, description = '''If true, import SCM configuration from the archive. 
 If false, do not import the SCM configuration (default).''', schema = @Schema(implementation = Boolean.class)),
+                    @Parameter(name = 'importComponents ', required = false, in = ParameterIn.QUERY, description = '''It imports components''', schema = @Schema(implementation = String.class)),
```

Comment:
it looks like this new parameter is intended to do the same as "importComponents.NAME"